login-workflow/src/new-architecture/contexts/AuthContext/dictionaries/i18n.ts

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import i18n from 'i18next';
+import { fr, enUS, es, zhCN, ptBR } from 'date-fns/locale';
+
+// for getting the device's language locale
+// 'en' is default deviceLocale
+
+const getDateLocale = (): Locale => {
+    switch (i18n.language) {
```

Comment:
This is duplicate code. We should migrate this to a central utils folder/file. Something like `src/utils/i18nUtils.ts` perhaps. Be sure to add an index file for the utils folder and export it up to the top level of new-architecture for now.